caom2-datalink-server/src/main/java/ca/nrc/cadc/caom2/datalink/ArtifactProcessor.java

```diff
@@ -179,6 +192,9 @@ else if (ReleaseType.META.equals(a.getReleaseType()))
                 readable = ar.metaReadable;
             // else: new releaseType is not likely without major caom design change
             
+            if (readable != null) {
+                pkgReadable = pkgReadable && readable;
```

Comment:
Don't think the pkgReadable in the condition is necessary but does no harm.  (ie could be: pkgReadable = readable)

Reply:
The idea here is that once pkgReadble become false (non readable artifact) it stays false.